com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -68,29 +68,10 @@ public void NetworkUpdate(NetworkUpdateStage updateStage)
         }
 
         /// <summary>
-        /// GetStreamBufferFrameCount
-        /// Returns how many frames have been processed (Inbound/Outbound)
-        /// </summary>
-        /// <param name="queueType"></param>
-        /// <returns>number of frames procssed</returns>
-        public uint GetStreamBufferFrameCount(RpcQueueHistoryFrame.QueueFrameType queueType)
-        {
-            return queueType == RpcQueueHistoryFrame.QueueFrameType.Inbound ? m_InboundFramesProcessed : m_OutboundFramesProcessed;
-        }
-
-        /// <summary>
-        /// AddToInternalMLAPISendQueue
-        /// NSS-TODO: This will need to be removed once we determine how we want to handle specific
-        /// internal MLAPI commands relative to RPCS.
-        /// Example: An network object is destroyed via server side (internal mlapi) command, but prior to this several RPCs are invoked for the to be destroyed object (Client RPC)
-        /// If both the DestroyObject internal mlapi command and the ClientRPCs are received in the same frame but the internal mlapi DestroyObject command is processed prior to the
-        /// RPCs being invoked then the object won't exist and additional warnings will be logged that the object no longer exists.
-        /// The vices versa scenario (create and then RPCs sent) is an unlikely/improbable scenario, but just in case added the CreateObject to this special case scenario.
-        ///
         /// To avoid the DestroyObject scenario, the internal MLAPI commands (DestroyObject and CreateObject) are always invoked after RPCs.
         /// </summary>
         /// <param name="queueItem">item to add to the internal MLAPI queue</param>
-        public void AddToInternalMLAPISendQueue(RpcFrameQueueItem queueItem)
+        internal void AddToInternalMLAPISendQueue(RpcFrameQueueItem queueItem)
```

Comment:
changes in this file are looking more like refactorings, should probably go into another isolated PR (and maybe not even targeting release branch IMHO)